### PR TITLE
 Added pyenv-offline-installer for offline installation.

### PR DESCRIPTION
**Pull request by huntzhan**
#28 related.

### CODE REVIEW

File: bin/pyenv-offline-installer [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+fi
+
+colorize() {
+  if [ -t 1 ]; then printf "\e[%sm%s\e[m" "$1" "$2"
+  else echo -n "$2"
+  fi
+}
+
+conditional_mv() {
+  [ -d "$2" ] || mkdir -p "$2" && mv "$1"/* "$2"
```
Comment (yyuu) [inline bin/pyenv-offline-installer]: Could you rewrite this with using if then fi? I think it'd be better to clarify the operator precedence.

**Comment by huntzhan** [comment]
Ops, sorry for my carelessness. Working on it.

**Comment by huntzhan** [comment]
Following code detects what kind of shell in which the script is running. I think most people would prefer to the automatically shell detection, instead of passing in the identifier of shell manually. So, could I just ignore such argument(the `shell`) and keeps `<path-to-archive>` as the only input arugment?

```sh
shell="$1"
if [ -z "$shell" ]; then
  shell="$(ps c -p "$PPID" -o 'ucomm=' 2>/dev/null || true)"
  shell="${shell##-}"
  shell="${shell%% *}"
  shell="$(basename "${shell:-$SHELL}")"
fi
```

**Comment by yyuu** [comment]
Umm. Basically I think it'd be better if the caller if user can override $shell configurations.

How do you think keep reading archive path from environment variable if specified, otherwise read it from stdin?

**Comment by huntzhan** [comment]
What about `$ PYENV_SHELL=zsh ./pyenv-offline-installer /path/to/archive.tgz` ?

**Comment by yyuu** [comment]
Argument handling should be common between online and offline installer. Sorry for confusion but just passing archive file via environment variable might be suitable for this....

**Comment by huntzhan** [comment]
get it.

**Pull request closed without merging**